src/GameState/Board.tsx

```diff
@@ -22,7 +21,12 @@ export const Board = (props: any) => {
               alignItems="center"
               justifyContent="center"
               key={cellIndex}
-              bg={marbleClickedCoordinates?.x === cellIndex && marbleClickedCoordinates?.y === rowIndex ? '#0bf220' : "#948e8b"}
+              bg={
+                marbleClickedCoordinates?.x === cellIndex &&
+                marbleClickedCoordinates?.y === rowIndex
+                  ? '#0bf220'
```

Comment:
I don't like to see colors in the code. They are not explicit, and it's difficult to build an harmonized app with flying colors :P